src/unk_0206A8DC.c

```diff
@@ -103,19 +103,19 @@ BOOL sub_0206A9B4 (UnkStruct_020507E4 * param0)
     return sub_0206A8EC(param0, (2400 + 5));
 }
 
-void sub_0206A9C4 (UnkStruct_020507E4 * param0)
+void Events_SetVsSeekerUsedFlag (UnkStruct_020507E4 * param0)
 {
     sub_0206A8DC(param0, (2400 + 13));
     return;
 }
 
-void sub_0206A9D4 (UnkStruct_020507E4 * param0)
+void Events_ClearVsSeekerUsedFlag (UnkStruct_020507E4 * param0)
 {
     sub_0206A8E4(param0, (2400 + 13));
     return;
 }
```

Comment:
These functions have unnecessary return statements.

Is it also possible to document or label the subroutines here? It seems like the subroutines under these (`sub_0205081C` and `sub_02050844`, respectively) turn a flag on and off, respectively.